spec/integration/prometheus_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../vcr/vcr_setup'
+require_relative '../spec_helper'
+
+module Hawkular::Prometheus::RSpec
```

Comment:
This is not used on the specs (it is in some places, but should be removed). The correct way to describe tests is:

```ruby
describe Hawkular::Prometheus do
  # tests
end
```

Reply:
If all spec files are using this pattern, then this is a refactoring.
Which is good, but I think it should go in a different PR not on this.

Reply:
Same as the other comment, this should be correct, given that it is new code. Another PR can be made to refactor the other ones.